docs/how-to/add-extra-context-to-logging.rst

```diff
@@ -0,0 +1,76 @@
+Logging
+=======
+
+Description
+-----------
+
+Here's how you can add extra context to your logging in Django-Socio-GRPC
+    
+Usage
+-----
+
+=====================
+Logging GRPC Services
+=====================
+
+You'll want to be able to log information about your grpc services.
+Django-Socio-GRPC is built-in with existing loggers to get information from GRPC services
+
+If you want to add information to your logs from grpc. You will need to reproduce something similar to these two functions.
+
+.. code-block:: python
+    def default_get_log_extra_context(service: "GenericService"):
+    """
+    It allow logs to have extra data about the current context of the log. Used especially for tracing system.
+    """
+    extra_context = {
+        "grpc_service_name": service.get_service_name(),
+        "grpc_action": service.action,
+    }
+    if hasattr(service.context, "user") and hasattr(service.context.user, "pk"):
+        extra_context["grpc_user_pk"] = service.context.user.pk
+    return extra_context
+
+
+    def set_log_record_factory():
```

Comment:
you don't explain how you have to call this method.
For information this method should be called before any log using the format and this should be explicitly documented IMO